terraform/SLI_quality_views.tf

```diff
@@ -0,0 +1,100 @@
+              census.tableId AS backup_table_id,
+              census.lastModifiedTime as backup_last_modified,
+              census.numBytes AS backup_num_bytes,
+              census.numRows AS backup_num_rows
+            FROM
+              [${var.bbq_project}.datastore_export_views_legacy.last_available_backup_for_every_table_entity]
+            AS last_backup
+            LEFT OUTER JOIN (
+              SELECT datasetId, tableId, lastModifiedTime, numBytes, numRows
+              FROM [${var.gcp_census_project}.bigquery_views_legacy_sql.table_metadata_v1_0]
+              WHERE projectId = "${var.bbq_project}"
+            ) AS census
+            ON census.datasetId=last_backup.backup_dataset_id AND census.tableId=last_backup.backup_table_id
+        EOF
+    use_legacy_sql = true
+  }
+
+  depends_on = ["google_bigquery_dataset.SLI_backup_quality_views_dataset"]
+}
+
+resource "google_bigquery_table" "SLI_quality" {
+  project = "${local.SLI_views_destination_project}"
+  dataset_id = "${var.SLI_backup_quality_views_dataset}"
+  table_id = "SLI_quality"
+
+  view {
+    query = <<EOF
+            #legacySQL
+            SELECT
+              source_table.projectId AS source_project_id,
+              source_table.datasetId AS source_dataset_id,
+              source_table.tableId AS source_table_id,
+              source_table.partitionId AS source_partition_id,
+              last_backup_in_census.backup_dataset_id AS backup_dataset_id,
+              last_backup_in_census.backup_table_id AS backup_table_id,
+              source_table.lastModifiedTime AS source_last_modified,
+              last_backup_in_census.backup_last_modified AS backup_last_modified,
+              source_table.numBytes AS source_num_bytes,
+              last_backup_in_census.backup_num_bytes AS backup_num_bytes,
+              source_table.numRows AS source_num_rows,
+              last_backup_in_census.backup_num_rows AS backup_num_rows
+            FROM
+              [${local.SLI_views_destination_project}.${var.SLI_backup_quality_views_dataset}.tables_not_modified_since_3_days]
+            AS source_table
+            LEFT JOIN (
+              SELECT
+                source_project_id, source_dataset_id, source_table_id, source_partition_id,
+                backup_dataset_id, backup_table_id, backup_last_modified, backup_num_bytes, backup_num_rows
+              FROM [${local.SLI_views_destination_project}.${var.SLI_backup_quality_views_dataset}.last_backup_in_census]
+            ) AS last_backup_in_census
+            ON source_table.projectId=last_backup_in_census.source_project_id AND
+               source_table.datasetId=last_backup_in_census.source_dataset_id AND
+               source_table.tableId=last_backup_in_census.source_table_id AND
+               source_table.partitionId=last_backup_in_census.source_partition_id
+            WHERE (source_table.numBytes-last_backup_in_census.backup_num_bytes) != 0
```

Comment:
source_table.numBytes  != last_backup_in_census.backup_num_bytes . instead of division

Reply:
Done